Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM mcr.microsoft.com/dotnet/aspnet:3.1 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
 WORKDIR /app
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/sdk:3.1 AS build
+FROM mmcr.microsoft.com/dotnet/sdk:6.0 AS build
```

Comment:
Typo?
`mmcr` vs `mcr`

Reply:
Hi @yzhoholiev,

Please turn on "Show all changes". Cause you are reviewing changes from intermediate commit. This typo was fixed by https://github.com/loic-sharma/BaGet/pull/717/commits/c58c40b0e8eed07360198b7afc957170820fb7e0